website/cue/reference/components/sinks/new_relic_logs.cue

```diff
@@ -60,7 +60,7 @@ components: sinks: new_relic_logs: {
 		requirements: []
 		warnings: [
 			"""
-				This transform has been deprecated. Please use the [`new_relic` sink](\(urls.vector_new_relic_sink)) instead.
+				This sink no longer exists as of 0.25.0. Please use the [`new_relic` sink](\(urls.vector_new_relic_sink)) instead.
```

Comment:
🤔 if it's been completely deleted from the code I expect we should delete the docs as well - I know we can redirect requests to the old docs to the new one.

Reply:
I'll have to look into our policy for this, but I imagine there might be a lot of people still using this sink given we just dropped it a few months ago?

Reply:
Fair, the website should be representative of the latest version of Vector. I _think_ anytime we've fully deleted a component we've removed the corresponding documentation.

cc @jszwedko

Reply:
FWIW the redirect can be done like [this](https://github.com/vectordotdev/vector/blob/master/website/content/en/docs/reference/configuration/sources/http_server.md?plain=1#L7).

Reply:
I went ahead and dropped these after checking in with @jszwedko. My newest commit adds the redirect. I still see new_relic_logs pages appear in the search results on the site, but the redirect is confirmed to work. I guess there's some cache bust or index to regenerate somewhere?

Reply:
😅 honestly I don't know how that algolia search works, from what I've seen it's generally just worked out after merges so I've never thought about it.